controller/volume_controller.go

```diff
@@ -3008,6 +3095,22 @@ func (vc *VolumeController) updateRecurringJobs(v *longhorn.Volume) (err error)
 	if err = datastore.FixupRecurringJob(v); err != nil {
 		return err
 	}
+
+	// DR volume will not restore volume recurring jobs/groups configuration.
+	if v.Spec.FromBackup != "" && !v.Spec.Standby {
```

Comment:
NIT: Similar here.
```
if v.Spec.FromBackup == "" || v.Spec.Standby {
    return nil
}
```

Reply:
Returning `nil` directly would not update the changes `datasotre.FixupRecurringJob` above.
Modified to 
```
if v.Spec.FromBackup != "" && !v.Spec.Standby && !v.Status.RestoreInitiated {
```

Reply:
I think you can make this check as a part of `shouldRestoreRecurringJobs`

Reply:
`v.Status.RestoreInitiated` will always be `true` when restoration is starting, and I want to restore recurring job only once.

Reply:
What if the restore volume is just created and hasn't been init?

Reply:
Moved this check into `shouldRestoreRecurringJobs`. Thanks.
Recurring jobs/groups will still be created if there is an error before `checkAndInitVolumeRestore` sets `RestoreInitiated` true, and it will skip creating recurring jobs/groups at next reconciliation